### ISSUE TITLE
Downloads not working

### ISSUE THREAD

**Issue by Kelador** [lusid1/psp#1]
Downloads on the site no longer work they just download a xpd file.

**Comment by lusid1** [comment]
Non-working downloads removed, for the most part.  Still looking for a fix for x-radar.
Fundamental issue is resources on archive are being bounced through an https/tls redirect with an unsupportable tls version.
